assets/js/blocks/product-gallery/frontend.tsx

```diff
@@ -9,14 +9,32 @@ interface State {
 
 interface Context {
 	woocommerce: {
-		productGallery: { numberOfThumbnails: number };
+		selectedImage: string;
+		imageId: string;
 	};
 }
 
 interface Selectors {
+	woocommerce: {
+		isSelected: ( store: unknown ) => boolean;
+	};
+}
+
+interface Actions {
+	woocommerce: {
+		productGallery: {
+			updateActiveProductGalleryImage: ( context: Context ) => void;
+		};
```

Comment:
Since we are not using the `productGallery` key anymore, I think we can remove it. 

We should also include the handleClick action here.

```suggestion
		handleClick: ( store: Store ) => void
```